README.md

```diff
@@ -264,10 +262,9 @@ In order to  remove a payment you will need the Invoice instance you want to upd
 ### Getting receipts
```

Comment:
Let's remove receipts from this README. They're no longer in our API docs, as we prefer our users to use the Invoices API.